subnet-scripts/bettensor/install.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+git clone https://github.com/bettensor/bettensor.git  ~/.bittensor/subnets/bettensor/
+cd ~/.bittensor/subnets/bettensor/ || exit
+
+sudo apt-get update && sudo apt-get install python3.10-venv
+
+python3.10 -m venv ~/venvs/bettensor
+source ~/venvs/bettensor/bin/activate
+
+if [ -f .env ]; then
+    export $(cat .env | grep -v '#' | awk '/=/ {print $1}')
+fi
+
+wandb login $WANDB_API_KEY
```

Comment:
```suggestion
wandb login "$WANDB_API_KEY"
```
ALWAYS quote variables in bash